### DISCUSSION TITLE
Add 'streak count' and 'max streak so far' on profile

### DISCUSSION THREAD

**Discussion by prathmesh796** [monkeytypegame/monkeytype#5799] [Ideas]
add the **current streak count** on the profile along with **maximum streak maintained so far** count. You can this similar feature in _doulingo, leetcode, chess.com_, etc.

**Comment by fehmer** [comment]
Current and max streak are already available on the profile page:
![image](https://github.com/user-attachments/assets/520e7cb1-1f87-40e6-9602-9589e33c3798)

**Reply by prathmesh796** [reply]
Actually, I can't see it on my account. Is there any settings to confiure it?? (I have a current one day streak)
![Screenshot 2024-08-18 170206](https://github.com/user-attachments/assets/82aae84b-c201-4588-954e-294637a3ca79)
